src/js/vic-v2/containers/ConfirmationPage.jsx

```diff
@@ -34,21 +37,39 @@ class ConfirmationPage extends React.Component {
     const response = form.submission.response
       ? form.submission.response.attributes
       : {};
+    const { veteranFullName, verified } = form.data;
+    const submittedAt = moment(form.submission.submittedAt);
```

Comment:
Could probably put most of this into a single object destructuring statement if you're into that kind of thing. I think you could even use a default for `form.submission.response.attributes` as an alternative to the ternary.

Reply:
I think it would look like:

`const { submittedAt, response: { attributes: response = {} } = {} } = form.submission;`

I find that harder to understand than the ternary, which maps more closely to the reason we're defaulting things: so that if if you go directly to this page without having submitted a form, you don't get errors because `response` is empty. I'll add a comment to the code though so it's more clear.